pkg/server/app/app.go

```diff
@@ -363,6 +364,33 @@ func (ops *AppOperations) List(user *database.User) ([]*AppListItem, error) {
 	return items, nil
 }
 
+func (ops *AppOperations) SetAutoScale(user *database.User, appName string, as *AutoScale) error {
```

Comment:
On `SetAutoScale`, S should be lower case. In general `autoscale` should be treated as a single word - this applies to all the occurrences throughout this PR.